core/src/main/java/com/microsoft/applicationinsights/TelemetryClient.java

```diff
@@ -385,6 +387,11 @@ public void trackPageView(PageViewTelemetry telemetry) {
      * @param telemetry The {@link com.microsoft.applicationinsights.telemetry.Telemetry} instance.
      */
     public void track(Telemetry telemetry) {
+
+        if (generateCounter.incrementAndGet() % 10000 == 0) {
+            InternalLogger.INSTANCE.info("Total events generated till now "+ generateCounter.get());
```

Comment:
Use format string here too (instead of string concat)

Reply:
Ah missed it. Thanks will do!